dlk/python/dlk/templates/include/quantizer.h

```diff
@@ -85,118 +46,16 @@ void func_QTZ_binary_mean_scaling(
   }
 }
 
-
-inline void func_QTZ_linear_mid_tread_half_body(
-  T_FLOAT input[],
-  T_INT nbit,
-  T_FLOAT max_value,
-  QUANTIZED_NOT_PACKED output[],
-  T_UINT begin,
-  T_UINT end)
-
-{
-  T_FLOAT max_value_r = 1.0f / max_value;
-
-  T_FLOAT min_value = 0.f;
-  T_FLOAT n = (1 << nbit) - 1.f;
-  int i = begin;
-
-#ifdef USE_NEON
-  float32x4_t max_value_x4 = vdupq_n_f32(max_value);
-  float32x4_t min_value_x4 = vdupq_n_f32(min_value);
-  float32x4_t round_offset = vdupq_n_f32(0.5);
-  float32x4_t max_value_rn = vdupq_n_f32(max_value_r * n);
-
-  for (; i <= static_cast<int>(end) - 4; i += 4)
-  {
-    float32x4_t tmp = vld1q_f32(&input[i]);
-    tmp = vmaxq_f32(tmp, min_value_x4);
-    tmp = vminq_f32(tmp, max_value_x4);
-    //    tmp = vmlaq_f32(tmp, max_value_rn, round_offset);
-    tmp = vmulq_f32(tmp, max_value_rn);
-    tmp = vaddq_f32(tmp, round_offset);
-    int32x4_t r = vcvtq_s32_f32(tmp);
-    int r_tmp[4];
-    vst1q_s32(r_tmp, r);
-    output[i] = r_tmp[0];
-    output[i+1] = r_tmp[1];
-    output[i+2] = r_tmp[2];
-    output[i+3] = r_tmp[3];
-  }
-#endif
-
-  for (; i < static_cast<int>(end); ++i)
-  {
-    T_FLOAT tmp = std::max(input[i], (T_FLOAT)min_value);
-    tmp = std::min(tmp, max_value);
-    output[i] = (T_INT)roundf(tmp * (max_value_r * n));
-  }
-}
-
-inline void func_QTZ_linear_mid_tread_half(
+void func_QTZ_linear_mid_tread_half(
     const TensorView<T_FLOAT, MemoryLayout::NHWC>& input,
     const TensorView<T_INT, MemoryLayout::Atom>& nbit,
     const TensorView<T_FLOAT, MemoryLayout::Atom>& max_value,
-    const TensorView<QUANTIZED_PACKED, MemoryLayout::HWChBCl>& output) {
-  Measurement::Start("QTZ_linear_mid_tread_half");
-
-  unsigned num_elems = input.size();
-  QUANTIZED_NOT_PACKED* output_not_packed = new QUANTIZED_NOT_PACKED[num_elems];
+    const TensorView<QUANTIZED_PACKED, MemoryLayout::HWChBCl>& output);
 
-  unsigned int chunk_size = num_elems / std::thread::hardware_concurrency();
-  if (chunk_size == 0) {
-    chunk_size += 1;
-  }
-
-  std::vector<std::thread> threads;
-  for (unsigned int i = 0; i < num_elems; i += chunk_size) {
-    threads.emplace_back(std::thread([&input, &nbit, &max_value, &output_not_packed, i, chunk_size, num_elems] {
-          func_QTZ_linear_mid_tread_half_body(input.data(), nbit(), max_value(), output_not_packed, i,
-                                              std::min(i + chunk_size, static_cast<unsigned int>(num_elems)));
-    }));
-  }
-
-  for (auto& th: threads) {
-    th.join();
-  }
-
-  //static T_UINT counter = 0;
-  //write_to_file("out/qconv_input_quantized_not_packed", counter++, output_not_packed, in_height * in_width * in_depth);
```

Comment:
These commented lines looks better to be removed.

Reply:
here is deleted.